packages/ai-core/tsconfig.cjs.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
[req] With this config the CJS config will be shipped when releasing. This should not be the case.